app/src/main/java/com/medmapper/v33001/room/MedicineRoomDatabase.kt

```diff
@@ -7,7 +7,7 @@ import androidx.room.RoomDatabase
 import com.medmapper.v33001.dao.MedicineDAO
 import com.medmapper.v33001.dto.Medicine
 
-@Database(entities = arrayOf(Medicine::class), version = 1, exportSchema = false)
+@Database(entities = [Medicine::class], version = 1, exportSchema = false)
```

Comment:
'arrayOf' call should be replaced with array literal [...]

Reply:
This change was covered in a previously submitted review.